src/test/java/roomescape/ReservationTest.java

```diff
@@ -0,0 +1,66 @@
+package roomescape;
+
+import static org.hamcrest.Matchers.is;
+
+import io.restassured.RestAssured;
+import io.restassured.http.ContentType;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.annotation.DirtiesContext;
+
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.DEFINED_PORT)
+@DirtiesContext(classMode = DirtiesContext.ClassMode.BEFORE_EACH_TEST_METHOD)
+public class ReservationTest {
+
+    @Test
+    @DisplayName("모든 예약을 조회한다.")
+    void getAllReservations() {
+        RestAssured.given().log().all()
+                .when().get("/admin/reservation")
+                .then().log().all()
+                .statusCode(200);
+
+        RestAssured.given().log().all()
+                .when().get("/reservations")
+                .then().log().all()
+                .statusCode(200)
+                .body("size()", is(0));
+    }
+
+    @Test
+    @DisplayName("예약에 대한 생성, 조회, 삭제를 진행한다.")
+    void reservationCrdr() {
+        Map<String, String> params = new HashMap<>();
+        params.put("name", "브라운");
+        params.put("date", "2023-08-05");
+        params.put("time", "15:40");
```

Comment:
[맵 팩토리](https://devbksheen.tistory.com/entry/%EB%AA%A8%EB%8D%98-%EC%9E%90%EB%B0%94-%EC%BB%AC%EB%A0%89%EC%85%98-%ED%8C%A9%ED%86%A0%EB%A6%ACCollection-Factory-%EC%86%8C%EA%B0%9C-%EB%B0%8F-%EC%82%AC%EC%9A%A9%EB%B2%95#%EB%A7%B5%20%ED%8C%A9%ED%86%A0%EB%A6%AC(Map%C2%A0Factory)-1)를 활용해봐도 좋을 것 같아요. 😃 
```suggestion
        Map<String, String> params = Map.of(
            "name", "브라운",
            "date", "2023-08-05",
            "time", "15:40"
        );
```

Reply:
컬렉션.of()를 컬렉션 팩토리라고 하는군요?!
링크까지 첨부해 주시고, 감사합니다!